use_exploring: false

footprint: [[0.35, 0.2],
            [-0.35, 0.2],
            [-0.35, -0.2],
            [0.35, -0.2]]

planner_patience: 3
sq_dist_for_goal_reached: 0.25
horizon_for_trajectory: 16

time_to_trigger_exploration: 9.0
time_to_trigger_replanning: 66.6

angular_limit_for_hgp: M_PI_8
xy_limit_for_hgp: 0.18

shutdown_costmaps: false
clearing_radius: 0.4
footprint_padding: 0.01

exploration_planner: hector_nav_core_exploration_plugin/HectorNavCoreExplorationPlugin
global_planner: hector_exploration_planner/HectorExplorationBaseGlobalPlannerPlugin
path_planner: SBPLLatticePlanner

recovery_behaviors: [{name: inverse_trajectory_recovery, type: inverse_trajectory_recovery/InverseTrajectoryRecovery}]

SBPLLatticePlanner:
  environment_type: XYThetaLattice
  planner_type: ARAPlanner
  allocated_time: 3.0
  initial_epsilon: 3.0
  forward_search: false
